release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -422,6 +428,21 @@ In the table, features are marked with the following statuses:
 AMD and Intel 64-bit architectures (x86-64-v2) will still be supported.
 ====
 
+[id="ocp-4-10-docker-config-location-deprecation"]
+==== Default Docker configuration location Deprecated
+Previously, `oc` commands that used a registry configuration obtained credentials from the Docker configuration location, which was `~/.docker/config.json` by default. This has been deprecated and will be replaced by a podman configuration location in a future version of {product-title}.
```

Comment:
I first tried to read this as "registry configuration-obtained", so a possible rewording for clarity could be:
```suggestion
Previously, `oc` commands that used a registry configuration would obtain credentials from the Docker configuration location, which was `~/.docker/config.json` by default. This has been deprecated and will be replaced by a Podman configuration location in a future version of {product-title}.
```